### PR TITLE
Make depth of 1 to be the default

### PR DESCRIPTION
**Pull request by sc68cal**

### CODE REVIEW

File: assets/in [hunk 1]
```diff
@@ -63,8 +63,8 @@ if [ -n "$branch" ]; then
   branchflag="--branch $branch"
 fi
 
-depthflag=""
-if test "$depth" -gt 0 2> /dev/null; then
+depthflag="--depth 1"
```
Comment (pivotal-jamie-klassen) [inline assets/in:66]:
```suggestion
depthflag="--depth $depth"
```
And then you don't need the `if` at all.

**Comment by sc68cal** [comment]
How do we get the DCO job to recheck? I force pushed a new commit with the added signoff

File: assets/in [hunk 2]
```diff
@@ -63,8 +63,8 @@ if [ -n "$branch" ]; then
   branchflag="--branch $branch"
 fi
 
-depthflag=""
-if test "$depth" -gt 0 2> /dev/null; then
+depthflag="--depth $depth"
+if test "$depth" -ne 1 2> /dev/null; then
   depthflag="--depth $depth"
 fi
```
Comment (pivotal-jamie-klassen) [inline assets/in:69]: Since both branches of the iff statement do the same thing, it can be dropped completely!
```suggestion
```

**Comment by sc68cal** [comment]
OK I guess I'll rebase & squash, and add you as a co-author, then force push. 🤷‍♂️

**Comment by pivotal-jamie-klassen** [comment]
@sc68cal the DCO check is still not passing. Is it possible github is doing something funny? When I look at the last two commits on your branch, they have no `Signed-off-by` line in the commit message, and in order for the check to pass they will need to contain

```
Signed-off-by: Sean M. Collins <sean@coreitpro.com>
```

sorry for the hassle!

**Comment by sc68cal** [comment]
rebased and added signoffs.

**Comment by sc68cal** [comment]
I'm actually going to fixup the commit messages that the github UI generated - I don't like them

**Comment by sc68cal** [comment]
ok, this should be good to merge

**Pull request merged**